controllers/helmrelease_controller_test.go

```diff
@@ -278,6 +281,208 @@ invalid`,
 	}
 }
 
+func TestValidation(t *testing.T) {
+	tests := []struct {
+		name       string
+		resources  []runtime.Object
+		references []v2.ValuesReference
+		values     string
+		wantErr    bool
+	}{
+		{
+			name: "valid ValuesKey",
+			resources: []runtime.Object{
+				valuesSecret("values", map[string][]byte{"single": []byte("true")}),
```

Comment:
This value never changes, how about putting this in a variable and reusing it in all the cases. Would be easy to update in the future if we need to.

Reply:
Or maybe this need not be in test cases and just be a static value.

Reply:
Resources are not required now that we are only validating the API level validations, and therefore has been removed.